test/utils/configUtils.js

```diff
@@ -27,7 +29,7 @@ function getExpectedFunctionConfig(options = {}) {
   return Object.assign({
     description: 'Serverless WarmUp Plugin',
     events: [{ schedule: 'rate(5 minutes)' }],
-    handler: '_warmup/index.warmUp',
+    handler: path.join('_warmup', 'index.warmUp'),
```

Comment:
As above, this was OK before. Since this is not a path to be used by the OS but a path used by Serverless.

Or am I missing something?